scripts/build/builders/gn.py

```diff
@@ -95,6 +95,8 @@ def _build(self):
         self.PreBuildCommand()
 
         cmd = ['ninja', '-C', self.output_dir]
+        if self.ninja_jobs > 0:
```

Comment:
0 is a valid value for ninja/make and means infinite number of jobs. The best way to define the `--ninja-jobs` option would be to allow using it just like `-j` with ninja, i.e.: `--ninja-jobs` (infinite jobs), `--ninja-jobs=0` or `--ninja-jobs=60`

Reply:
Done.